### PR TITLE
Add badges for license/cocoapods/travis ci

### PR DESCRIPTION
**Pull request by bootstraponline**
[ci skip]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # EarlGrey
+[![Apache License](https://img.shields.io/badge/license-Apache%202-lightgrey.svg?style=flat)](https://github.com/google/EarlGrey/blob/master/LICENSE)
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
```
Comment (khandpur) [inline README.md]: Do we filly support cartage without your pending PR which adds cartage spec?

Comment (bootstraponline) [inline README.md]: That's a good point. Carthage is fully supported but only if you know how to manually configure the testing target which is a complex and painful process. I'll remove the Carthage badge for now.

**Comment by bootstraponline** [comment]
Updated.

File: README.md (hunk 1 above)
Comment (tirodkar) [inline README.md]: Yup. We should add a Carthage badge only when we provide a "Carthage Installation Steps" guide in the docs.

**Comment by khandpur** [comment]
Thanks for adding these badges.

**Pull request merged**